src/edu/duke/cs/osprey/restypes/InterResBondingTemplate.java

```diff
@@ -59,9 +59,7 @@
     //atom is from this residue; can it bond other residues?
 
 	public abstract boolean isInterResBondedForward(Residue res1, Residue res2);
-	public abstract boolean makeInterResBondForward(Residue res1, Residue res2);
```

Comment:
TL:DR; This is probably safe to remove.

Interesting story here... looks like this function is no longer used, so it's probably safe to remove. It used to be used by the AtomConnectivity cache, back when it used a pre-population strategy. The cache used to generate all the possible dipepetides and cache their atom connectivity, so it needed a way to construct molecular geometry. But eagerly populating the cache could be slow in some cases because the code often generated too many dipeptides that would never get used. So now the cache only gets populated when the entries are needed, so we only needs query methods now.

Reply:
Yeah, I noticed it wasn't used in Java or Python so nuked it.